.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Run lint & unit tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    name: "Run unit tests"
+    runs-on: [buildjet-2vcpu-ubuntu-2204-arm]
+    container:
+      image: ghcr.io/viamrobotics/ocean-prefilter:arm64
+      options: --platform linux/arm64
+
+    steps:
+    - uses: actions/checkout@v4
+
+
+
+    - name: Run lint
+      run: |
+        make lint
+
```

Comment:
If you already lint in the other github action, you don't need to lint here as well